src/index.html

```diff
@@ -5,8 +5,386 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+     rel="stylesheet"
+     >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <img
+        class="header-logo"
+        src="./images/logo.png"
+        alt="MOYO"
+        >
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+            class="header__nav-link
+            is-active"
+            href="#apple"
+            >
+            Apple
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              data-qa="nav-hover"
+              href="#laptops&computers"
+            >
+             Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#photo">Photo</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+
+    <!-- #region product cards -->
+    <main class="cards">
+      <div class="card" data-qa="card">
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+        >
+
+        <div class="card__info">
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__reviews">
+            <ul class="stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+
+            <p class="card__reviews-count">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <p class="card__price-amount">$2,199</p>
+          </div>
+
+          <a href="#"
+            class="card__btn"
+            data-qa="hover"
+          >
+            Buy
+          </a>
```

Comment:
Need to fix it everywhere
```suggestion
          <a
            href="#"
            class="card__btn"
            data-qa="hover"
          >
            Buy
          </a>
```